website/docs/d/cloudant.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cloudant"
+description: |-
+  Get information about Cloudant instance.
+subcategory: "Cloud Databases"
+---
+
+# ibm_cloudant
+
+Provides a read-only data source for an existing IBM Cloud Cloudant service. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+resource "ibm_resource_instance" "cloudant" {
+    name     = "cloudant-service-name"
+    service  = "cloudantnosqldb"
+    plan     = "lite"
+    location = "us-south"
+ }
+
+ data "ibm_cloudant" "instance" {
+    name     = ibm_resource_instance.cloudant.name
+ }
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `id` (Optional, String) The unique identifier of the Cloudant resource.
+* `location` (Optional, String) The location or the environment in which instance exists.
+* `name` (Required, String) The name of the IBM Cloudant resource instance.
+* `resource_group_id` (Optional, String) The id of the resource group in which the instance is present. If not provided it takes the default resource group.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `capacity` (Number) A number of blocks of throughput units. For more details please read about [`blocks`](https://cloud.ibm.com/apidocs/cloudant#putcapacitythroughputconfiguration) parameter.
+* `cors_config` (List of Object) Configuration for CORS. (see [below for nested attributes](#nestedatt--cors_config))
```

Comment:
delete (see [below for nested attributes](#nestedatt--cors_config)) and replace as 
  Nested scheme for `cors_config`: